### PR TITLE
Rename App stylesheet and JS file to cbdebugger

### PR DESCRIPTION
**Pull request by homestar9**
This pull requests helps prevent Elixir naming conflicts with App.scss and App.js files which are commonly used in web apps.  Cbdebugger will now use cbdebugger.scss and cbdebugger.js

### CODE REVIEW

File: layouts/includes/footer.cfm [hunk 1]
```diff
@@ -9,7 +9,7 @@
 		manifestRoot : args.manifestRoot
 	)#"></script>
 	<script async="async" defer="defer" src="#event.getModuleRoot( "cbDebugger" )##html.elixirPath(
-		fileName : "js/App.js",
+		fileName : "js/Cbdebugger.js",
```
Comment (lmajano) [inline layouts/includes/footer.cfm:12]: can you just use `cbdebugger.js` all lowercase.

File: layouts/includes/head.cfm [hunk 2]
```diff
@@ -1,7 +1,7 @@
 <cfoutput>
 	<!---css --->
 	<link href="#event.getModuleRoot( "cbdebugger" )##html.elixirPath(
-		fileName : "css/App.css",
+		fileName : "css/Cbdebugger.css",
```
Comment (lmajano) [inline layouts/includes/head.cfm:4]: can you use all lowercase `cbdebugger.css`

File: webpack.config.js [hunk 3]
```diff
@@ -46,8 +46,8 @@ module.exports = elixir( mix => {
 
 	// Mix App styles
 	mix
-		.js( "App.js" )
-		.sass( "App.scss" )
+		.js( "Cbdebugger.js" )
```
Comment (lmajano) [inline webpack.config.js:49]: Same as above

**Review by lmajano — CHANGES_REQUESTED** [review]

File: layouts/includes/footer.cfm (hunk 1 above)
Comment (homestar9) [reply layouts/includes/footer.cfm]: Agreed. I like lowercase as well.

File: webpack.config.js (hunk 3 above)
Comment (homestar9) [reply webpack.config.js]: Agreed. I like lowercase as well.

File: layouts/includes/head.cfm (hunk 2 above)
Comment (homestar9) [reply layouts/includes/head.cfm]: Agreed. I like lowercase as well.

**Pull request merged**